index.js

```diff
@@ -11,6 +16,12 @@ module.exports = function vueify (file) {
   function dependency(file) {
     stream.emit('file', file)
   }
+
+  function style(buf) {
+    if (config.cssExport) {
+      fs.appendFile(config.cssExport, buf)
```

Comment:
I think `cssExport mode` would only come in handy for production mode build. Anyhow, I'm finding `vueify` isn't as simple as one would want so I'm trying to use Vue with ES6 template strings for (lol, pun) defining Vue templates and [css-ns](https://github.com/jareware/css-ns) for simple style scoping.

I'm not saying vueify is not useful, but I'm not sure if it's a good idea to implement something that other tools already support.

If you agree, please close this issue.